frontend/dashboard/pages/PageLayout/DashboardHeader/DashboardHeader.tsx

```diff
@@ -0,0 +1,135 @@
+import React, { useContext } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { useTranslation } from 'react-i18next';
+import {
+  StudioAvatar,
+  StudioPageHeader,
+  StudioProfileMenu,
+  useMediaQuery,
+  type StudioProfileMenuItem,
+} from '@studio/components';
+import { useSelectedContext } from 'dashboard/hooks/useSelectedContext';
+import { type Organization } from 'app-shared/types/Organization';
+import { repositoryBasePath, repositoryOwnerPath } from 'app-shared/api/paths';
+import { MEDIA_QUERY_MAX_WIDTH } from 'app-shared/constants';
+import { HeaderContext, SelectedContextType } from 'dashboard/context/HeaderContext';
+import { useLogoutMutation } from 'app-shared/hooks/mutations/useLogoutMutation';
+import { getOrgNameByUsername, getOrgUsernameByUsername } from 'dashboard/utils/userUtils';
+
+const TRUNCATE_APP_USERNAME = 30;
```

Comment:
Veldig fint at du har tallet i en egen variabel, men det er fortsatt ikke helt klart for meg hva dette tallet brukes til. Å legge til enhet i variabelnavnet hjelper som regel mye.